juntagrico/dao/jobdao.py

```diff
@@ -61,3 +61,7 @@ def get_pinned_jobs():
     def get_promoted_jobs():
         return RecuringJob.objects.filter(type__name__in=Config.promoted_job_types(),
                                           time__gte=timezone.now()).order_by('time')[:Config.promomted_jobs_amount()]
+
+    @staticmethod
+    def upcomming_jobs_for_member(member):
+        return Job.objects.filter(time__gte=timezone.now(), assignment__member=member).distinct()
```

Comment:
Why is it possible to access 'assignment' from Job? I don't see any relationship in the entity classes.

Reply:
short answer magic, the long answer ist that django creates reversable field in the entity of a foreign key. More about that here
https://docs.djangoproject.com/en/2.2/topics/db/queries/#backwards-related-objects

Reply:
Cool. It makes sense now!